docs/advanced/symbol-server-compatibility.md

```diff
@@ -297,7 +303,7 @@ The following layout types support this lookup:
 If you have no requirements to be compatible with another system you can also
 use the "unified" directory layout structure. This has the advantage that it's
 unified across all platforms and thus easier to manage. It can store breakpad
-files, PDBs, PEs and everything else.  The `symsorter` tool in the symbolicator
+files, PDBs, PEs and everything else. The `symsorter` tool in the symbolicator
```

Comment:
😮

Reply:
The actual issue here is that someone refuses to double space on fullstop in the rest of the documentation. WONTFIX!

Reply:
:)